### PR TITLE
Latest version with most of api finished

### PR DESCRIPTION
**Pull request by panosatha**

### CODE REVIEW

File: flood_adapt/integrator/sfincs_adapter.py [hunk 1]
```diff
@@ -21,11 +22,11 @@ def __init__(self, model_root: Optional[str] = None):
         # if validate_existence_root_folder(model_root):
         #    self.model_root = model_root
 
-        self.sf_model = SfincsModel(root=model_root, mode="r+")
+        # self.sf_model = SfincsModel(root=model_root, mode="r+")
```
Comment (GundulaW) [inline flood_adapt/integrator/sfincs_adapter.py:25]: Do we have a fix for the hydromt issue yet?

We only need to comment out the import of the SfincsModel from hydromt. The rest can remain in the script. At least this worked for me

File: flood_adapt/object_model/interface/database.py
Comment (GundulaW) [inline flood_adapt/object_model/interface/database.py:1]: projection and event methods (get/save/edit/delete/copy) are still missing

**Review by GundulaW — APPROVED** [review]
1. Do we have a fix for the hydromt issue yet?
We only need to comment out the import of the SfincsModel from hydromt. The rest can remain in the script. At least this worked for me

2. The get/save/edit/delete/copy methods are missing in the database interface class, should be quick to fix

File: flood_adapt/integrator/sfincs_adapter.py (hunk 1 above)
Comment (panosatha) [reply flood_adapt/integrator/sfincs_adapter.py:25]: we install the correct hydromt version with the environment yml file to make sure that it works.

**Pull request merged**